Apps/phcybereason/cybereason_connector.py

```diff
@@ -612,6 +612,60 @@ def _handle_set_reputation(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def upgradeSensor(self, ActionParams):
+    
```

Comment:
1. Rename function as _handle_upgradeSensor()
2. Add docstring comments for public function